core/includes/install.core.inc

```diff
@@ -361,8 +361,8 @@ function install_run_tasks(&$install_state) {
  * Runs an individual installation task.
  *
  * @param $task
- *   An array of information about the task to be run.
- * @param $install_state
+ *   An array of information about the task to be run as returned by
+ *   hook_install_tasks(). * @param $install_state
```

Comment:
Oops, looks like `@param $install_state` should be on a separate line. Could you correct and push a fix?